core/src/main/kotlin/org/lflang/generator/ts/TSParameterPreambleGenerator.kt

```diff
@@ -202,7 +202,7 @@ class TSParameterPreambleGenerator(
         |    if (__processedCLArgs.id !== null) {
         |        __federationID = __processedCLArgs.id;
         |    } else {
-        |        Log.global.error(__clUsage);
+        |        Log.error(null, () => __clUsage);
```

Comment:
This looks a little weird, but I think it's our own API. What is the `null` argument for?

Reply:
When `message` (arg1) is called, it is called as if it is a function of `object` (arg0), and `arg0` will/might be its `this`.
If no `this` is referenced, then passing in `null` would suffice.

We might need to refactor this later, since this design is a bit unnecessary.

https://github.com/lf-lang/reactor-ts/blob/9798f65d700d2e801c2a5c242b32704f74e47d72/src/core/util.ts#L70-L84